### PR TITLE
Modify S5344: Re-arrange the folder for new languages

### PR DESCRIPTION
**Pull request by loris-s-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S5344/common/fix/plaintext-password.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (egon-okerman-sonarsource) [inline rules/S5344/common/fix/plaintext-password.adoc:1]: Nitpick: remove this line

**Review by gaetan-ferry-sonarsource — APPROVED** [review]
LGTM

File: rules/S5344/common/fix/plaintext-password.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+==== Never store passwords in cleartext
```
Comment (loris-s-sonarsource) [inline rules/S5344/common/fix/plaintext-password.adoc:2]:
```suggestion
==== Never store passwords in cleartext

```

File: rules/S5344/common/fix/plaintext-password.adoc [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+A user password should never be stored in clear text. Instead, a hash should be
+produced from it using a secure algorithm. When dealing with password storage
+security, best practices recommend relying on a slow hashing algorithm, that
+will make brute force attacks more difficult. Using a hashing function with
+adaptable computation and memory complexity also is recommended to be able to
+increase the security level with time.
+
+Adding a salt to the digest computation is also recommended to prevent
+pre-computed table attacks (see rule S2053).
+
```
Comment (loris-s-sonarsource) [inline rules/S5344/common/fix/plaintext-password.adoc:12]:
```suggestion
```

File: rules/S5344/common/fix/plaintext-password.adoc [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+==== Never store passwords in cleartext
+A user password should never be stored in clear text. Instead, a hash should be
```
Comment (egon-okerman-sonarsource) [inline rules/S5344/common/fix/plaintext-password.adoc:4]: plain-text and clear text/cleartext are used interchangeably within this rule text. Please choose one and replace all other instances

File: rules/S5344/common/fix/password-hashing.adoc [hunk 5]
```diff
@@ -0,0 +1,10 @@
+==== Use specific password hashing algorithms
+
+In general, relying on an algorithm with no known weaknesses is also
```
Comment (egon-okerman-sonarsource) [inline rules/S5344/common/fix/password-hashing.adoc:3]: I'm not sure this sentence makes sense anymore in the order that the paragraphs are displayed now

**Review by egon-okerman-sonarsource — COMMENTED** [review]
A few small changes are necessary

File: rules/S5344/common/fix/password-hashing.adoc (hunk 5 above)
Comment (loris-s-sonarsource) [reply rules/S5344/common/fix/password-hashing.adoc:3]: ah yeah you are right, I blindly modified this

**Pull request merged**